juniper_official/Kernel/check-pfeman-conn-drops.rule

```diff
@@ -0,0 +1,69 @@
+/*
+ * Peer infra module is responsible for replicating FIB states to the line cards
+ * and also to process incoming IPCs from them. Each line card has a connection
+ * with Master Routing Engine Junos Kernel.
+ *
+ * This rule monitors and ungraceful connection drops in these pfeman connections.
+ */
+iceberg {
+    topic kernel.peer-infra {
+        synopsis "Monitor peer-infra activities on the system";
+        rule check-pfeman-conn-drops {
+            sensor kernel-peer-infra-sensor {
+                open-config {
+                    sensor-name /junos/kernel/peer-infra;
+                    frequency 5m;
+                }
+            }
+            field peer-pfeman-conn-drops {
+                sensor kernel-peer-infra-sensor {
+                    path /junos/kernel/peer-infra/peer-pfeman-conn-drops;
+                }
+                type integer;
+                description "Number of pfeman connection drops";
+            }
+            trigger report-pfeman-conn-drops {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed